content/en/providers/check-in/sp/_index.md

```diff
@@ -291,99 +278,17 @@ value pairs for the Claims.
 The following scope values can be used to request Claims from the EGI
 Check-in UserInfo Endpoint:
 
+| Scope                          | Claims                                                                                                    |
+| ------------------------------ | --------------------------------------------------------------------------------------------------------- |
+| `openid`                       | `sub`                                                                                                     |
+| `profile`                      | - `name` <br> - `given_name` <br> - `family_name` <br> - `preferred_username`                             |
```

Comment:
maybe try with a <ul> instead of <br>? Not really sure what's best for maintenance of the docs / rendering

Reply:
> Just one thing on how to format the lists inside the tables as I'm not really sure on what's the best approach, @gwarf, any opinion here?

So by definition html code in markdown is not great (and making linters to complain)(yes we can disable some options, but it's suboptimal) and should be avoided as much as possible, but unfortunately markdown is also a bit limited for the advanced management of tables.

So in case we have not other option than using HTML, maybe using some tags containing more meanings would be more appropriate, like:
 ```html
<ul><li>Userinfo endpoint</li><li>introspection endpoint</li></ul>
```

Also when `<br>` are required it would be preferable to use the `<br />` syntax.

Reply:
Thanks @nikosev!